pkgs/top-level/python-packages.nix

```diff
@@ -859,6 +859,23 @@ in {
 
   kconfiglib = callPackage ../development/python-modules/kconfiglib { };
 
+  todoman = disabledIf (!isPy3k) callPackage ../applications/office/todoman {
```

Comment:
IIRC:

* `python-packages` can't include anything outside of pkgs/development/python-modules
* `pythonPackages` is for python modules, not for applications

Reply:
OK so how will I be able to write this in a way that satisfies:

- vdirsyncer is available as a module in `python3.pkgs` so `khal` will be able to use it from the same python packages set used to build it while it's also available as an application.
- The python version should be override-able for all of these utilities.

I'm considering using [toPythonPackage](https://nixos.org/nixpkgs/manual/#topythonmodule-function) instead but it seems it's only appropriate for bindings where in the case of `vdirsyncer` it's an almost pure python library.

Reply:
`toPythonApplication`

Reply:
OK. In the latest changes I used `toPythonPacakge` only for `vdirsyncer` which is imported as a python module in `khal`'s tests. @veprbl what's all packages by `@pimutils` considered an application, are override-able with `python3 = python38;` and specifically vdirsyncer with `python38` can be used via: `python38.pkgs.vdirsyncer`.